src/overlays/actors/ovl_En_Horse_Zelda/z_en_horse_zelda.c

```diff
@@ -186,34 +187,34 @@ void func_80A6DC7C(EnHorseZelda* this) {
     Animation_PlayOnce(&this->skin.skelAnime, sAnimationHeaders[this->animationIndex]);
 }
 
-void func_80A6DCCC(EnHorseZelda* this, PlayState* play) {
+void EnHorseZelda_Stop(EnHorseZelda* this, PlayState* play) {
     this->actor.speedXZ = 0.0f;
     if (SkelAnime_Update(&this->skin.skelAnime)) {
-        func_80A6DC7C(this);
+        EnHorseZelda_SetupStop(this);
     }
 }
 
-void func_80A6DD14(EnHorseZelda* this) {
-    f32 sp34;
+void EnHorseZelda_SetupGallop(EnHorseZelda* this) {
+    f32 speedMod;
 
     this->action = 1;
     this->animationIndex = 0;
-    sp34 = this->actor.speedXZ / 6.0f;
+    speedMod = this->actor.speedXZ / 6.0f;
     Audio_PlaySfxGeneral(NA_SE_EV_HORSE_RUN, &this->actor.projectedPos, 4, &gSfxDefaultFreqAndVolScale,
                          &gSfxDefaultFreqAndVolScale, &gSfxDefaultReverb);
     Animation_Change(&this->skin.skelAnime, sAnimationHeaders[this->animationIndex],
-                     splaySpeeds[this->animationIndex] * sp34 * 1.5f, 0.0f,
+                     splaySpeeds[this->animationIndex] * speedMod * 1.5f, 0.0f,
                      Animation_GetLastFrame(sAnimationHeaders[this->animationIndex]), ANIMMODE_ONCE, 0.0f);
 }
 
-void func_80A6DDFC(EnHorseZelda* this, PlayState* play) {
-    func_80A6D918(this, play);
+void EnHorseZelda_Gallop(EnHorseZelda* this, PlayState* play) {
+    EnHorseZelda_Move(this, play);
     if (SkelAnime_Update(&this->skin.skelAnime)) {
-        func_80A6DD14(this);
+        EnHorseZelda_SetupGallop(this);
```

Comment:
Perhaps this is just me, but when I see `SetupGallop` I imagine it is preparing everything needed for `Gallop`, and than sets `Gallop` as the next action function. That doesn't seem what is going on here so seems a little confusing to me. 
Perhaps `Spur/Spurred` instead, as it seems to be called when the horse gets spurred to gallop again?